agentendpoint/exec_task.go

```diff
@@ -184,7 +184,7 @@ func (e *execTask) run(ctx context.Context) error {
 		}
 	case agentendpointpb.ExecStepConfig_POWERSHELL:
 		if goos == "windows" {
-			exitCode, err = executeCommand(winPowershell, []string{"-File", localPath}, e.LogLabels)
+			exitCode, err = executeCommand(winPowershell, []string{"-ExecutionPolicy", "RemoteSigned", "-File", localPath}, e.LogLabels)
```

Comment:
Just set this to Bypass, 
also now that I look at this use these flags as well:
"-NonInteractive", "-NoProfile"

Reply:
Done. I'll update API comment and gcloud help text to mention these accordingly